internal/integrationtests/scripts/e2e_integration_test.sh

```diff
@@ -91,7 +104,7 @@ for config_name in "${options[@]}"; do
   done
 
   # Example of using the exported variables
-  DESCRIPTION="$platform - $DISTRIBUTION_ACCOUNT_TYPE - $REGISTRATION_CONTACT_TYPE"
+  DESCRIPTION="$platform$CIRCLE_API_TYPE - $DISTRIBUTION_ACCOUNT_TYPE - $REGISTRATION_CONTACT_TYPE"
```

Comment:
```suggestion
  DESCRIPTION="$platform - $CIRCLE_API_TYPE - $DISTRIBUTION_ACCOUNT_TYPE - $REGISTRATION_CONTACT_TYPE"
```
Or did you mean to have this as `CircleTRANSFERS`, `CirclePAYOUTS` ?

Reply:
> Or did you mean to have this as `CircleTRANSFERS`, `CirclePAYOUTS` ?

Yes, that's what I meant, and it will be empty for non-circle cases.